include/openPMD/IO/AbstractIOHandler.hpp

```diff
@@ -57,6 +57,12 @@ class unsupported_data_error : public std::runtime_error
     {}
 };
 
+enum class FlushMode : bool
```

Comment:
As discussed:
https://openpmd-api.readthedocs.io/en/0.14.4/details/mpi.html

Flush is always declared collective.